src/whining.js

```diff
@@ -0,0 +1,68 @@
+// Description:
+//   Whining
+//
+//   A hubot script that creates an agenda for Wednesday Whining
+//
+// Configuration:
+//   HUBOT_IDIO_WHINING_BRANCH: optional
+//   HUBOT_IDIO_WHINING_ORG: optional
+//   HUBOT_IDIO_WHINING_REPO: optional
+//   HUBOT_IDIO_WHINING_TOKEN: required
+//
+// Commands:
+//   hubot it's whine time - Push a whining agenda to the docs repo
+//
+// Author:
+//   Mal Graty
+
+'use strict'
+
+const Octokat = require('octokat')
+const octo = new Octokat({token: process.env.HUBOT_IDIO_WHINING_TOKEN})
+
+const branch = process.env.HUBOT_IDIO_WHINING_BRANCH || 'master'
```

Comment:
Do we want this to write straight to master, or do we want it to create a branch named after the date, and simply open up the PR which the chair can freely merge after whining.

Reason being that changes in schedule or whatever means someone can put an additional commit on the branch prior to merging rather than people dicking about directly committing to master